doc/source/conf.py

```diff
@@ -46,7 +47,10 @@
 
 # General information about the project.
 project = 'pyqtgraph'
-copyright = '2011 - {}, Luke Campagnola'.format(datetime.now().year)
+now = datetime.utcfromtimestamp(
```

Comment:
I noticed a note in the [`datetime.utcfromtimestamp`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp) docs that `datetime.fromtimestamp(timestamp, timezone.utc)` may be preferred to `utcfromtimestamp`. It seems like the `year` attribute would be safe from the issues they're warning against, so I'd defer to Debian folks on this -- if this is the usual approach for python projects, I'm fine with it. 

Also, it might be good to have a brief comment above this line indicating that getting the date in this way is for reproducible builds. Just a link to [SOURCE_DATE_EPOCH](https://reproducible-builds.org/specs/source-date-epoch/) would probably suffice. I see it's already in the commit message, but I think having it here in the code wouldn't hurt.

Reply:
This is how I've been fixing it for a long time now with no issue, if that helps. :+1: